ui/src/shared/components/lastRunTaskStatus/LastRunTaskStatus.tsx

```diff
@@ -0,0 +1,80 @@
+// Libraries
+import React, {FC, useRef, useState} from 'react'
+import classnames from 'classnames'
+
+// Components
+import {
+  Icon,
+  IconFont,
+  Popover,
+  Appearance,
+  PopoverInteraction,
+  ComponentColor,
+} from '@influxdata/clockface'
+
+// Styles
+import './LastRunTaskStatus.scss'
```

Comment:
is this where we want to import this? or should it be imported in in [chronograf.scss](https://github.com/influxdata/influxdb/blob/master/ui/src/style/chronograf.scss) ?

Reply:
@hoorayimhelping this was input by @alexpaxton. I'm not entirely sure on best practices but if it's the case that it should be input into the chronograf file I can go ahead and do that